src/main/webapp/about.html

```diff
@@ -68,13 +67,21 @@ <h6>As an Interviewer</h6>
       <h6>As an Interviewee</h6>
       <p>
         To search for an interview, navigate to the <a href="search-interview.html">
-        Scheduling Page</a>. You can choose what position you are interested in and see what times 
+        Scheduling Page</a>. Specify that you are searching as an interviewee. You can then choose what position you are interested in and see what times 
         and interviewers are available. You will be able to see their company and job, but no more 
         personal information.
         <br>
         Once an interviewer is matched with you, you will get more details in an email. The 
         interviews will be one hour long and hosted on Google Meet.
-      </p> 
+      </p>
+      <h6>As an Shadow</h6>
+      <p>
+        To search for an interview, navigate to the <a href="search-interview.html">
+        Scheduling Page</a>. Specify that you are searching as a shadow. You can then choose what position you are interested in and see what existing interviews you can shadow. After selecting a time, you will be able to see the company and job of the interviewer, but no more personal information.
```

Comment:
Long line. Also, are we planning on sending emails? Let's put a TODO in ScheduledInterviewServlet if so.

Reply:
l'll make an issue in Git. 
Update description to say long lines will be formatted after content is approved.